common.mk

```diff
@@ -142,15 +141,34 @@ FIRTOOL_TARGETS = \
 	$(FIRTOOL_SMEMS_CONF) \
 	$(FIRTOOL_TOP_SMEMS_JSON) \
 	$(FIRTOOL_TOP_HRCHY_JSON) \
-	$(FIRTOOL_MODEL_MOD_HRCHY_JSON) \
+	$(FIRTOOL_MODEL_HRCHY_JSON) \
```

Comment:
It might be good to have a linter go over the Makefile to make sure nothing is broken (maybe look into this pre-commit hook: https://github.com/mrtazz/checkmake#pre-commit-usage)

Reply:
Sure, will do.